### PR TITLE
feat: [Commands] `lang:find` show bad keys when scanning (v2)

### PR DESCRIPTION
**Pull request by neznaika0**
**Description**
Superseds #8149 

- Add filepath in list bad keys
- Add natural sorting output values (bad keys)

```console
$ ./spark lang:find --verbose

...

Files found: 10
New translates found: 30
Bad translates found: 5
+------------------------+---------------------------------+
| Bad Key                | Filepath                        |
+------------------------+---------------------------------+
| ..invalid_nested_key.. | app/Controllers/Translation.php |
| .invalid_key           | app/Controllers/Translation.php |
| TranslationBad         | app/Controllers/Translation.php |
| TranslationBad.        | app/Controllers/Translation.php |
| TranslationBad...      | app/Controllers/Translation.php |
+------------------------+---------------------------------+

All operations done!
```
**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Review by michalsn — COMMENTED** [review]
I may not get something here... This command is described as:
> Find and save available phrases to translate

Why does it suddenly show the wrong language keys? And how does it know which keys are bad?

**Comment by neznaika0** [comment]
Have you read the previous issue? @kenjis and I thought it would be worthwhile to warn the user when scanning that there are incorrect keys.
It still does its function. Only additionally notifies about it

Look at the output. Which keys are bad?
- They do not start with the file name (dots). 
- Have duplicate dots at the beginning\at the end. 
- They have only the file name (one line, without dots).

In short, anything that does not fall under the correct ``File.key``, ``File.nested.key`` format

**Review by michalsn — APPROVED** [review]
Ok, now I get it. Thanks.

**Review by kenjis — APPROVED** [review]

File: system/Commands/Translation/LocalizationFinder.php [hunk 1]
```diff
@@ -172,14 +186,16 @@ private function findTranslationsInFile($file): array
         preg_match_all('/lang\(\'([._a-z0-9\-]+)\'\)/ui', $fileContent, $matches);
```
Comment (neznaika0) [inline system/Commands/Translation/LocalizationFinder.php:193]: There is one feature: The regular expression initially discards bad keys. They will not be shown in the list.

**Comment by kenjis** [comment]
@neznaika0 Thank you!
